test/models/registry/api.test.ts

```diff
@@ -0,0 +1,14 @@
+import { getCertifiedRealmInfo } from 'models/registry/api'
+import { getConnectionContext } from 'utils/connection'
+import realms from 'public/realms/mainnet-beta.json'
+
+test('getCertifiedRealmInfo', async () => {
+  const mango = realms.find(({ symbol }) => symbol === 'MNGO')!
+
+  const realmInfo = await getCertifiedRealmInfo(
+    mango.symbol,
+    getConnectionContext('mainnet')
+  )
+
+  expect(realmInfo!.realmId.toBase58()).toEqual(mango.realmId)
+})
```

Comment:
I will be replacing this test with more useful ones but this is just to ensure that I can run tests without loading any frontend/next-specific code

Reply:
:+1: